### PR TITLE
Add support to specify which translation files are analyzed

### PR DESCRIPTION
**Pull request by robinborst95**
In our application we have English translations as a base language and all other languages are maintained externally on Phrase (so that those translations do not rely on developers speaking the language). When English translations are added or removed, they are pushed to Phrase, where they are added to or removed from the other languages, and then those changes are pulled back into the source code again. However, we recently ran into a problem with that workflow when we tried to remove an English translation, because this addon marked that translation as unused in the other languages. Although this is not incorrect (it is in fact unused), we don't want to remove those translations manually, but instead remove them from Phrase which removes them from the source code when they're pulled in (which is a separate step in our development process).
A similar issue occurs when an English translation is removed by accident, with the translation still being present in the other languages. Then that translation is not seen as missing, because it's still there in the other languages.

Long story short, it would really help if we can configure which translation files are analyzed. Because of these issues we've removed this analyze step from our CI workflow to prevent developers having a failing build when we don't want to.

In our case the config value of `translationFiles` would be conditional, so something like `process.env.ONLY_ANALYZE_EN_TRANSLATIONS ? ['**/en-us.yaml'] : null`, but that felt too specific to include in the readme.

### CODE REVIEW

**Comment by robinborst95** [comment]
@Mikek2252 I can't assign reviewers and I don't know if you have time soon, but would you have time to review this? For our application this would be really valuable, so I hope this is worth releasing soon :pray:

**Comment by Mikek2252** [comment]
@robinborst95 Hey, Im fine with the changes, but would it be possible to update the readme to be a little less specific? such as `if you would like to only analyse a sub set of translations files you can use...` i think this will be a little clearer on whether it is something they would want to use.

**Comment by robinborst95** [comment]
I've simplified the readme now, let me know what you think :+1:

**Review by Mikek2252 — APPROVED** [review]

**Comment by Mikek2252** [comment]
If i'm honest i dont think i quite understand, but that is separate from this PR so i'll merge this and create a release

**Comment by robinborst95** [comment]
Many thanks :pray:
